Webview API guide: TypeScript error in the `dispose()`example

The second code block in the [Lifecycle](https://code.visualstudio.com/api/extension-guides/webview#lifecycle) section showing an example usage of the `dispose()` function contains the following error:
```
src/extension.ts:20:46 - error TS2345: Argument of type 'string' is not assignable to parameter of type '"Coding Cat" | "Compiling Cat"'.

panel.webview.html = getWebviewContent(cats['Coding Cat']);
                                       ~~~~~~~~~~~~~~~~~~
Found 1 error. Watching for file changes.
```

--- a/api/extension-guides/webview.md
+++ b/api/extension-guides/webview.md
@@ -279,7 +279,7 @@ export function activate(context: vscode.ExtensionContext) {
         {}
       );
 
-      panel.webview.html = getWebviewContent(cats['Coding Cat']);
+      panel.webview.html = getWebviewContent('Coding Cat');
 
       // After 5sec, programmatically close the webview panel
       const timeout = setTimeout(() => panel.dispose(), 5000);

